test/functional/commands/distribute/Release.Tests.ps1

```diff
@@ -8,8 +8,7 @@ Describe "distribute release" {
 
   It "creates a release when mandatory flag is set to true" {
     # Arrange
-    $fileName = "Dummy.zip"
-    "DummyData" | Out-File -FilePath $fileName
+    $fileName = "./test/functional/dummyapk.apk"
```

Comment:
Considering the app created here is a custom app (not android), why did we need to upload a valid APK? This does not sound right, as there is no metadata extraction for custom apps.

Reply:
Addressed in #928